vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -7210,7 +7341,24 @@ unit:FT2-PER-BTU_IT-IN
   qudt:definedUnitOfSystem sou:IMPERIAL ;
   qudt:definedUnitOfSystem sou:USCS ;
   qudt:expression "\\(ft2-per-btu-in\\)"^^qudt:LatexString ;
-  qudt:hasDimensionVector qkdv:A0E0L-1I0M-1H1T3D0 ;
+  # unit:FT2-PER-BTU_IT-IN has incorrect dimension vector:
+  # 	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H1T3D0, which should be
+  # 	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H0T2D0
+  # factor unit tree for unit ft²/btu⋅in
+  #  ──┬ unit:FT2-PER-BTU_IT-IN        	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H1T3D0
+  #    ├─── unit:BTU_IT^-1                	http://qudt.org/vocab/dimensionvector/A0E0L-2I0M-1H0T2D0
+  #    ├─── unit:FT^2                     	http://qudt.org/vocab/dimensionvector/A0E0L2I0M0H0T0D0
+  #    └─── unit:IN^-1                    	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M0H0T0D0
+  #
+  # the unit has these quantityKinds: ThermalResistivity
+  #  ══╤ qk:ThermalResistivity
+  #    ├─── unit:DEG_F-HR
+  #    ├─── unit:FT2-PER-BTU_IT-IN
+  #    ├─── unit:K-M-PER-W
+  #    └─── unit:M-K-PER-W
+  # quantity kinds that fit the corrected dimension vector http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H0T2D0, and associated units:
+  # (none found)
```

Comment:
I would opt for qk:Unknown. A web search did not turn up anything. We could make a qk:AreaPerEnergyLength or something like that, but I don't know what that means. (however, we have >100 qk:Unknown units, and making new QKs for them would be a separate task)

Reply:
done